README.md

```diff
@@ -311,6 +315,14 @@ common use case for this feature is to launch a set of containers
 in parallel, e.g. for CI builds. Container prefixes can also be supplied by the
 `CRANE_PREFIX` environment variable.
 
+#### Override image tag
+Using a global `--tag` flag overrides image tag parts of containers. If you
+specify `--tag 2.0.0-rc2`, an image name `repo/app:1.0` is treated as
+`repo/app:2.0.0-rc2`. Crane also accepts the `CRANE_TAG` environment variable as
+a default value of `--tag`. Note `--tag` is given priority over `CRANE_TAG`.
+Thus some hooks that rely on the `CRANE_TAG` environment variable may not work
+correctly if you specify `--tag` and `CRANE_TAG` at the same time. To avoid
+confusion, consistent use of `CRANE_TAG` is preferable.
```

Comment:
This sounds a bit complicated. Would it be possible to set `CRANE_TAG` if `--tag` is given?